.github/workflows/CI.yaml

```diff
@@ -51,4 +51,15 @@ jobs:
               aws s3 rm $dest
               aws s3 sync output $dest
 
+      - name: Update gh-pages
+        run: |
+             if [ $GITHUB_REF = "refs/heads/master" ]; then
+                git config --global user.email "$(git log -1 --format=%ae)"
+                git config --global user.name "$(git log -1 --format=%an)"
+                git subtree add --prefix=our-site origin gh-pages
+                cp -r  output/self/* our-site
```

Comment:
Shouldn't we first remove all files in `our-site`, to get rid of outdated files?

Reply:
good point, I will do the followup